bolt-simple-taproot.md

```diff
@@ -1163,137 +1231,117 @@ A new TLV stream is added to the `revoke_and_ack` message:
+- MUST fail the channel if `next_local_nonces` is absent, or cannot be parsed.
+- MUST fail the channel if an active commitment funding txid is missing in
+  `next_local_nonces`.
+- if `channel_reestablish` includes a `next_funding_txid` which matches the
+  latest interactive funding transaction or the current channel funding transaction
+  - if it must re-transmit its `commitment_signed`
+    - if `current_commit_nonce` is missing: MUST fail the channel
 
-A node:
-
-  - If ALL of the following conditions apply:
-  
-    - It has previously sent a `commitment_signed` message
-  
-    - It never processed the corresponding `revoke_and_ack` message
-  
-    - It decides to retransmit the exact `update_` messages from the last sent
-      `commitment_signed`
-  
-  - THEN it must regenerate the partial signature using the newly received
-    `next_local_nonce`
 
 ### Splice Coordination
 
 Splicing allows parties to modify the funding output of an existing channel without
 closing it. During splice operations, multiple commitment transactions may exist
-concurrently, each requiring its own MuSig2 nonce coordination. The `local_nonces`
-field enables this coordination by mapping transaction IDs to their respective nonces.
+concurrently, each requiring its own MuSig2 nonce coordination. The `next_local_nonces`
+field enables this coordination by mapping funding transaction ids to their 
+respective nonce.
 
 #### Splice Nonce Management
 
 During splice negotiation:
 
-- Each splice transaction MUST have a unique TXID as the key in the `local_nonces` map
-- The primary (non-splice) commitment transaction MUST use an empty hash (32 zero
-  bytes) as its key
-- Parties MUST include nonces for all active splice transactions in their
-  `local_nonces` map
+- Parties MUST include nonces for all active commitment transactions in their
+  `next_local_nonces` map, indexed by their funding tx id.
 - Completed or abandoned splices SHOULD have their nonces removed from the map in
   subsequent messages
 
 ##### Requirements for Splice Coordination
 
 When a splice is initiated:
 
-- The initiating party MUST generate a fresh nonce for the splice transaction
-- Both parties MUST add the splice TXID and corresponding nonce to their
-  `local_nonces` map
-- The nonce MUST be communicated in the next `commitment_signed` or
-  `channel_reestablish` message
-
+- nonces are added to `tx_complete`, including a specific, random 
+  `funding_nonce` used to spend the previous funding transaction.
+- nonces for each active commitment transaction are added to `revoke_and_ack`
+  and `channel_reestablish`.
+- 
 When a splice is completed:
 
 - Parties SHOULD remove the splice TXID from their `local_nonces` map
 - The primary commitment nonce SHOULD be updated to reflect the new funding output
 
 When multiple splices are pending:
 
-- Each splice MUST have a distinct TXID and nonce pair
-- Nonces MUST NOT be reused across different splice transactions
```

Comment:
I think generally more changes ahve been made to this section that needed. Here we have clear guidelines when multiple splices are pending, but it's been removed in place of an informal sentence.